src/vma/dev/rfs_uc.cpp

```diff
@@ -153,26 +153,21 @@ bool rfs_uc::prepare_flow_spec()
 
 bool rfs_uc::rx_dispatch_packet(mem_buf_desc_t* p_rx_wc_buf_desc, void* pv_fd_ready_array)
 {
-	// Dispatching: Notify new packet to the FIRST registered receiver ONLY
+	uint32_t num_sinks = (safe_mce_sys().enable_socketxtreme ?
```

Comment:
Pkease avoid using safe_mce_sys at the fast path

Reply:
My experiments did not show difference in results comparing `safe_mce_sys().enable_socketxtreme` usage  and special class filed in this function. So made a decision to use this way to avoid cloning the variable.